app/grandchallenge/ai_website/models.py

```diff
@@ -0,0 +1,170 @@
+    hq = models.CharField(max_length=100)
+    email = models.EmailField(max_length=500)
+    logo = models.ImageField(
+        upload_to=get_logo_path, storage=public_s3_storage, null=True,
+    )
+    description = models.CharField(
+        max_length=500,
+        default="",
+        blank=True,
+        help_text="Short summary of this project, max 500 characters.",
+    )
+    description_short = models.CharField(
+        max_length=250,
+        blank=True,
+        help_text="Short summary of this project, max 250 characters.",
+    )
+
+    def __str__(self):
+        return self.company_name
+
+
+class ProductImage(models.Model):
+    img = models.ImageField(
+        upload_to=get_images_path, storage=public_s3_storage
+    )
+
+
+# Create your models here.
+class ProductEntry(models.Model):
+    STATUS_CERTIFIED = "cer"
+    STATUS_YES = "yes"
+    STATUS_NO = "no"
+    STATUS_NA = "na"
+    STATUS_CLEARED = "cle"
+    STATUS_DE_NOVO_CLEARED = "dnc"
+    STATUS_PMA_APPROVED = "pma"
+    STATUS_UNKNOWN = "unk"
+
+    VERFIFIED_CHOICES = ((STATUS_YES, "Yes"), (STATUS_NO, "No"))
+
+    CE_STATUS_CHOICES = (
+        (STATUS_CERTIFIED, "Certified"),
+        (STATUS_NO, "No or not yet"),
+        (STATUS_NA, "Not applicable"),
+        (STATUS_UNKNOWN, "Unknown"),
+    )
+
+    FDA_STATUS_CHOICES = (
+        (STATUS_CLEARED, "510(k) cleared"),
+        (STATUS_DE_NOVO_CLEARED, "De novo 510(k) cleared"),
+        (STATUS_PMA_APPROVED, "PMA approved"),
+        (STATUS_NO, "No or not yet"),
+        (STATUS_NA, "Not applicable"),
+        (STATUS_UNKNOWN, "Unknown"),
+    )
+
+    ICONS = {
+        STATUS_CERTIFIED: "icon_check.png",
+        STATUS_YES: "icon_check.png",
+        STATUS_NO: "icon_no.png",
+        STATUS_NA: "icon_na.png",
+        STATUS_CLEARED: "icon_check.png",
+        STATUS_DE_NOVO_CLEARED: "icon_check.png",
+        STATUS_PMA_APPROVED: "icon_check.png",
+        STATUS_UNKNOWN: "icon_question.png",
+    }
+
+    created_date = models.DateField(default=timezone.now)
+    modified_date = models.DateField(default=timezone.now)
+    published_date = models.DateField(blank=True, null=True)
+    product_name = models.CharField(max_length=200)
+    company = models.ForeignKey(
+        CompanyEntry, on_delete=models.CASCADE
+    )  # product.company.company_name
+    short_name = models.CharField(
+        max_length=500,
+        blank=False,
+        help_text=(
+            "short name used in url, specific css, files etc. No spaces allowed"
+        ),
+        unique=True,
+    )
+    description = models.CharField(
+        max_length=400,
+        default="",
+        blank=True,
+        help_text="Short summary of this project, max 300 characters.",
+    )
+    description_short = models.CharField(
+        max_length=250,
+        default="",
+        blank=True,
+        help_text="Short summary of this project, max 250 characters.",
+    )
+    modality = models.CharField(max_length=100)
```

Comment:
We have this model in challenges already and an m2m field could be used here instead.